types/jquery/jquery-tests.ts

```diff
@@ -6103,11 +6103,17 @@ function JQuery() {
         }
 
         function parents() {
-            // $ExpectType JQuery<HTMLElement>
+            // $ExpectType JQuery<HTMLSpanElement>
             $('p').parents('span');
 
             // $ExpectType JQuery<HTMLElement>
             $('p').parents();
+
+            // $ExpectType JQuery<HTMLDivElement>
+            $(document).find('select').parents('div');
```

Comment:
This might seem redundant, but while `$('select')` doesn't return a `JQuery<HTMLSelectElement>`, `find('select')` does.